lxd/storage_btrfs.go

```diff
@@ -271,7 +254,20 @@ func (s *storageBtrfs) ContainerSnapshotCreate(
 func (s *storageBtrfs) ContainerSnapshotDelete(
 	snapshotContainer container) error {
 
-	return s.ContainerDelete(snapshotContainer)
+	err := s.ContainerDelete(snapshotContainer)
+	if err != nil {
+		return fmt.Errorf("Error deleting snapshot %s: %s", snapshotContainer.NameGet(), err)
+	}
+
+	oldPathParent := filepath.Dir(snapshotContainer.PathGet(""))
+	shared.Log.Debug(
+		"Trying to remove the parent path",
+		log.Ctx{"container": snapshotContainer.NameGet(), "parent": oldPathParent})
```

Comment:
Shouldn't that be in the if statement below? Printing this every time a snapshot is removed seems a bit excessive.